spec/app/ics-020-fungible-token-transfer/README.md

```diff
@@ -49,7 +49,7 @@ interface FungibleTokenPacketData {
 
 As tokens are sent across chains using the ICS 20 protocol, they begin to accrue a record of channels for which they have been transferred across. This information is encoded into the `denom` field. 
 
-The ics20 token denominations are represented the form `{ics20Port}/{ics20Channel}/{denom}`, where `ics20Port` and `ics20Channel` are an ics20 port and channel on the current chain for which the funds exist. The prefixed port and channel pair indicate which channel the funds were previously sent through. If `{denom}` contains `/`, then it must also be in the ics20 form which indicates that this token has a multi-hop record. Note that this requires that the `/` (slash character) is prohibited in non-IBC token denomination names.
+The ics20 token denominations are represented the form `{ics20Port}/{ics20Channel}/{denom}`, where `ics20Port` and `ics20Channel` are an ics20 port and channel on the current chain for which the funds exist. The prefixed port and channel pair indicate which channel the funds were previously sent through. Implementations are responsible for correctly parsing the IBC trace information from the base denomination. Since the channel identifiers are all auto-generated in the form: `channel-n`. ibc-go uses this fact to correctly parse out the IBC trace information from the base denom which may have slashes, but will not have a prefix: `transfer/channel-n`. If this assumption is broken, the trace information will be parsed incorrectly. Thus chains must make sure that base denominations do not have the ability to create arbitrary prefixes that can mock the ICS20 logic.
```

Comment:
```suggestion
The ics20 token denominations are represented the form `{ics20Port}/{ics20Channel}/{denom}`, where `ics20Port` and `ics20Channel` are an ics20 port and channel on the current chain for which the funds exist. The prefixed port and channel pair indicate which channel the funds were previously sent through. Implementations are responsible for correctly parsing the IBC trace information from the base denomination. Since the channel identifiers are all auto-generated in the form: `channel-n`. ibc-go uses this fact to correctly parse out the IBC trace information from the base denom which may have slashes, but will not have a substring: `transfer-port-name/channel-n`. If this assumption is broken, the trace information will be parsed incorrectly. Thus chains must make sure that base denominations do not have the ability to create arbitrary prefixes that can mock the ICS20 logic.
```
Main point of confusion is that it isn't about the prefix but the substring when splitting the slashes evenly. See [here](https://github.com/cosmos/ibc-go/blob/main/modules/apps/transfer/types/trace.go#L97). Okay with leaving `transfer` instead of `transfer-port-name`, just a technicality that on ibc-go it could be a different value and still parse fine